src/components/ButtonDropdown/ButtonDropdown.stories.tsx

```diff
@@ -59,3 +61,36 @@ export const PositionBottomRight: StoryObj<Args> = {
     position: 'bottom-right',
   },
 };
+
+const TestComponent = ({ color }: { color: string }) => {
+  return (
+    <>
+      <DropdownMenuItem>
+        <Icon name="schedule" purpose="decorative" size="1.25rem" />
+        Item 1
+      </DropdownMenuItem>
+      <DropdownMenuItem>
+        <div style={{ backgroundColor: color }}>
+          <Icon name="schedule" purpose="decorative" size="1.25rem" />
+          Item 2
+        </div>
```

Comment:
Is the thing that differentiates this story from the ones above that it has these `div`s wrapping the `DropdownMenuItem` contents and the different colored backgrounds? I don't see any other differences.

Reply:
Yuh the point of this story was just to have a helper component wrapping the fragment. This was the cause of the bug previously and this story tests for it.

Reply:
Could you leave some kind of comment about that? I could see a future dev not realizing the significance of this test and removing it for seeming redundant.

Reply:
Oooh yea good call, will do